README.rst

```diff
@@ -87,6 +87,8 @@ or with ``--help`` as so::
     Password for 'username' in 'system':
     $ keyring get system username
     password
+    $ keyring list app_name
```

Comment:
Probably this should mention "system" and not "app_name", because "system" is what's mentioned above.